### DISCUSSION TITLE
How to name the queries in the API explorer?

### DISCUSSION THREAD

**Discussion by papi1992** [mirumee/ariadne#1159] [General]
hi everyone,

could anyone help me in how I name the queries in the API explorer?

<img width="369" alt="image" src="https://github.com/mirumee/ariadne/assets/8364680/c1ce19cc-c1a4-4b83-9bf4-66bf18aa685f">

I'm talking about the untitled.

thanks in advance!

**Comment by rafalp** [comment]
Start your GraphQL operation with `query Name` or `mutation Name` or `subscription Name`.
